### PR TITLE
Feat/user api

### PR DESCRIPTION
**Pull request by silverware13**

### CODE REVIEW

File: src/components/ViewPlan.js [hunk 1]
```diff
@@ -44,12 +46,10 @@ export default class ViewPlan extends React.Component {
...
+        userId = obj[0][0].studentId;
         this.setState({
           courses: obj
         });
-        this.setState({
-          studentName: "Luke Skywalker"
-        });
         this.setState({
           planName: obj[0][0].planName
         });
```
Comment (clairecahill) [inline src/components/ViewPlan.js:55]: This can be done in one "setState" function

File: src/components/ViewPlan.js [hunk 2]
```diff
@@ -70,6 +70,26 @@ export default class ViewPlan extends React.Component {
...
+        // we got a bad status code
+        try {
+          throw response;
+        } catch (err) {
+          err.text().then(errorMessage => {
+            alert(errorMessage);
+          });
+        }
+      }
+
```
Comment (clairecahill) [inline src/components/ViewPlan.js:92]: The alert "no plan was found" was given 3 times...I'm not sure if this is the part of the code that executes that but we should probably look into it

**Review by clairecahill — COMMENTED** [review]
Overall it looks really good, I think you should change the setState to be only one call to the function and then it can be merged

File: src/components/ViewPlan.js (hunk 2 above)
Comment (silverware13) [reply src/components/ViewPlan.js:92]: I think that is due to changes I made in merge and not here.

**Pull request merged**